clang/runtime/dpct-rt/include/dpct/blas_gemm_utils.hpp

```diff
@@ -44,6 +44,19 @@ using matmul_desc_ptr = matmul_desc_t *;
 class transform_desc_t;
 using transform_desc_ptr = transform_desc_t *;
 
+template <typename primitive_type, typename... args_type>
+inline
+typename primitive_type::primitive_desc dgelu_epilogue_(
```

Comment:
The function name looks like a type name, please refine it.

Reply:
OK